testcases/kernel/syscalls/mmap/mmap04.c

```diff
@@ -147,7 +147,7 @@ static void setup(void)
 	}
 
 	/* Write test buffer contents into temporary file */
-	if (write(fildes, tst_buff, page_sz) < page_sz) {
+	if (write(fildes, tst_buff, page_sz) < (long)page_sz) {
```

Comment:
Technically write returns ssize_t. I guess that it actually has the same size as long as far as I can tell, but it would be a bit more future proof to have ssize_t there instead.